pkg/util/util.go

```diff
@@ -19,6 +19,9 @@ const (
 	HostRootMount                             = "/proc/1/root/"
 	CPUManagerOS3Path                         = HostRootMount + "var/lib/origin/openshift.local.volumes/cpu_manager_state"
 	CPUManagerPath                            = HostRootMount + "var/lib/kubelet/cpu_manager_state"
+
+	PCIResourcePrefix  = "PCI_RESOURCE"
+	MDevResourcePrefix = "MDEV_PCI_RESOURCE"
```

Comment:
Usually, if it is not clear where to place an entity, it ends up in some `util` package.
I do not think this is a good idea, it causes many parts of the codebase to depend on things that are irrelevant (e.g. all source code files that depend on this `util` got effected due to this change, but most have nothing to do with PCI or MDEV I suspect).

Reply:
Do you have suggestion on where it should be moved then?

Reply:
@VirrageS I think staging/src/kubevirt.io/api/core/v1/types.go would be a better place for these contacts.

Reply:
Thanks @vladikr. Moved the constants to `types.go`.